internal/core/services/service.go

```diff
@@ -0,0 +1,150 @@
+package services
```

Comment:
And this may be renamed to `app`, not a basis service

Reply:
Fixed, renamed.